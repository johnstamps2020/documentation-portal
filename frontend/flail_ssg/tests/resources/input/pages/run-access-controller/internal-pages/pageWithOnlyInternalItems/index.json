{
  "$schema": "/frontend/page-schema.json",
  "title": "Page with only internal items",
  "template": "page",
  "class": "threeCards",
  "selector": {
    "label": "Select page",
    "selectedItem": "Page with only internal items",
    "items": [
      {
        "label": "Page with only internal docs",
        "page": "../pageWithOnlyInternalDocs"
      },
      {
        "label": "Page with mixed items",
        "page": "../pageWithMixedItems"
      }
    ]
  },
  "items": [
    {
      "label": "Intermediate page with only internal items",
      "page": "intermediatePageWithOnlyInternalItems"
    },
    {
      "label": "Cloud Console Guide",
      "id": "supersecretgwcloudconsoleguideinsurerdev"
    }
  ]
}
