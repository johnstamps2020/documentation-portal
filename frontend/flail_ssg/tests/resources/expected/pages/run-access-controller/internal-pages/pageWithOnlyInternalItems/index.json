{
  "$schema": "/frontend/page-schema.json",
  "title": "Page with only internal items",
  "template": "page",
  "class": "threeCards",
  "selector": {
    "label": "Select page",
    "selectedItem": "Page with only internal items",
    "items": [
      {
        "label": "Page with only internal docs",
        "page": "../pageWithOnlyInternalDocs",
        "public": false,
        "internal": true
      },
      {
        "label": "Page with mixed items",
        "page": "../pageWithMixedItems",
        "public": false,
        "internal": false
      }
    ]
  },
  "items": [
    {
      "label": "Intermediate page with only internal items",
      "page": "intermediatePageWithOnlyInternalItems",
      "public": false,
      "internal": true
    },
    {
      "label": "Cloud Console Guide",
      "id": "supersecretgwcloudconsoleguideinsurerdev",
      "public": false,
      "internal": true
    }
  ],
  "public": false,
  "internal": true
}
